### PR TITLE
austin.intro.2.king.joe

### PR DESCRIPTION
**Pull request by josephwilliamking**
first commit

### CODE REVIEW

File: 02LessonTwo/in-class-project/index.html [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+      <tr>
+        <th><img src="https://pbs.twimg.com/profile_images/638327600486719488/Z-1qM3qs_400x400.jpg"/></th>
+          <td>yoga</td>
+        <th><img src="http://static.wixstatic.com/media/f0df91_41c602d5d2154bfb94bf248939a18d01.jpg_256"/></th>
+          <td>cycling</td>
+        <th><img src="https://pbs.twimg.com/profile_images/595624096861331457/cYqHxMGn.jpg"/></th>
+          <td>crossfit</td>
+      </tr>
+    </table>
+
```
Comment (fasteddiegarcia) [inline 02LessonTwo/in-class-project/index.html:25]: This is great looking code; organized and thorough. You need a way to contact your company here and the paragraph about the company should be at least 4 sentences long. All in all, great job.

File: 03LessonThree/in-class-project/daily-planet.html [hunk 2]
```diff
@@ -1,10 +1,59 @@
...
+                            <img src="https://cdn3.iconfinder.com/data/icons/social-icons-5/607/Twitterbird.png"> </td>
+                        <td>
+                            <a href="mailto:test@test.com"> <img src="http://www.freeiconspng.com/uploads/email-icon-23.png" </td>
+                                <td>
+                                    <a href="https://www.instagram.com/?hl=en"> <img src="http://3835642c2693476aa717-d4b78efce91b9730bcca725cf9bb0b37.r51.cf1.rackcdn.com/Instagram_App_Large_May2016_200.png" </td>
+        </table>
+    </footer>
+</body>
+
+</html>
```
Comment (ethanstenis) [inline 03LessonThree/in-class-project/daily-planet.html:59]: Great job! 

You are missing one of the project requirements - Add copyright and company name to footer. If you can add and recommit, I can bump your grade up 5 points.

File: 11LessonEleven/in-class-project/index.html [hunk 3]
```diff
@@ -1,10 +1,39 @@
...
+        <div class="right">
+            <div class="colorgreen">
+            </div>
+            <h1>Title</h1>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+        </div>
+    </main>
 </body>
+
 </html>
```
Comment (ethanstenis) [inline 11LessonEleven/in-class-project/index.html:39]: Please fix the layout here and I can bump your grade up.

screen shot 2016-09-13 at 5 42 22 pm

File: 14LessonFourteen/in-class-project/css/style.css [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+.header a {
+    float: right;
+    margin-right: 25px;
+    margin-top: -8px;
+    padding-left: 20px;
+    text-decoration: none;
+    color: white;
+}
+.hero {
+    background-image: url("../img/wine-bg.jpg");
```
Comment (zacacollier) [inline 14LessonFourteen/in-class-project/css/style.css:24]: be careful with your [relative path notation](https://css-tricks.com/quick-reminder-about-file-paths/). The way you have this path prefixed (`"../`) is pointing the browser to whatever folder is one step above `/img/`. That's why your images aren't loading. Use `"./` instead.

**Pull request closed without merging**